server/services/reportSummary.ts

```diff
@@ -70,14 +70,22 @@ const createUseOfForceDetails = (
   }
 }
 
+const getRelocationType = relocationType => {
+  return relocationType ? ` - ${toLabel(RelocationType, relocationType).toLowerCase()}` : ''
+}
+
+const checkRelocationType = relocationAndInjuries => {
```

Comment:
maybe `getNonCompliancyReason` would be a better method name?

Reply:
Implemented as suggested.

Reply:
done